Tests/PhishingDetectionTests/PhishingDetectionDataStoreTests.swift

```diff
@@ -61,6 +61,22 @@ class PhishingDetectionDataStoreTests: XCTestCase {
         XCTAssertEqual(actualHashPrefix, expectedHashPrefix)
     }
 
+    func testWhenEmbeddedRevisionNewerThanOnDisk_ThenLoadEmbedded() async {
```

Comment:
You should set the dataStore sets and revision to something else as well  and then check which one of the two wins…
At the moment if loadRevisionFromDisk() < dataProvider.embeddedRevision { this line of code is not called at all from the test.

Also I would recommend to check both cases disk higher and embedded higher